packages/cra-template/package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "@fluentui/cra-template",
-  "version": "0.2.1",
+  "version": "8.0.0",
```

Comment:
hmm this doesn't make much sense from semver perspective nor from functionality perspective. 

I'd suggest to bump it to `0.3.0` as your PR contains  fix to local tooling and bump of dependency to v8.

Most importantly from CRA template functionality there is nothing major breaking.

Reply:
I mentioned this in the description, I think it makes sense for the template to use the same major version as the main package. That way if we bump the main package version, it's easy to tell which template goes with which version, or even to use the previous major version's template if you need an app with that version for some reason. The normal usage/meaning of semver is less relevant for a template package since you only use it once, when the app is created (there's no continued dependency).